### DISCUSSION TITLE
Is it possible to update the Prompt after creation.

### DISCUSSION THREAD

**Discussion by cosmez** [waf/PrettyPrompt#234] [Q&A] [answered]
in the documentation you could update the prompt by specifying a string on ReadLineAsync

```csharp
var response = await prompt.ReadLineAsync("> ");
```

but now you have to configure the prompt text with `PromptConfiguration` and the text property is readonly 

```csharp
public FormattedString Prompt { get; }
```

is there a way to change it after? i would like to change the prompt depending on the state of my application.

**Comment by kindermannhubert** [comment, marked as answer]
That's reasonable and should be possible. I've created issue #235 for it.
